File: source/src/models/requests/PagingRequest.ts
class PagingRequest {
    PageIndex: int = 1;
    PageSize: int = 25;
    Filter: string = null;
    Prefix: string = null;

    constructor(pageIndex: int, pageSize: int, filter: string, prefix: string) {
        this.PageIndex = pageIndex;
        this.PageSize = pageSize;
        this.Filter = filter;
        this.Prefix = prefix;
    }
}

export default PagingRequest;

File: back-end/TT.StoreManager.Model/Request/PagingRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TT.StoreManager.Model
{
    public class PagingRequest
    {
        public int PageIndex { get; set; } = 1;
        public int PageSize { get; set; } = 25;
        public string Filter { get; set; }
    }
}
